src/data-services/regions.js

```diff
@@ -10,11 +10,11 @@ export function fetchRegions() {
         version: '1.0.0',
         service: 'WFS',
         request: 'GetFeature',
-        typeName: 'bc_regions:bc-regions-polygon',
+        typeName: 'bc_regions:BC-regions-FNLF-84',
```

Comment:
FNLF-84 is first nation language families? What's 84?

Reply:
84 is a hint for the WGS 84 CRS of the shapefile, matching the CRS of the BC regions polygon to which the new regions were appended.